### DISCUSSION TITLE
VLAN Konfiguration in ar7.cfg

### DISCUSSION THREAD

**Discussion by fischli** [Freetz-NG/freetz-ng#200] [Allgemein]
Lassen sich VLANs in der ar7.cfg konfigurieren oder besteht die einzige Möglichkeit VLANs zu nutzen über angepasste rc.custom Scripte? Hat jemand Erfahrungen im Einsatz von VLANs mit FRITZ!Boxen?

**Comment by fda77** [comment]
Evlt hilft das: https://freetz-ng.github.io/freetz-ng/wiki/50_Security/switch_config.html Wie weit es noch zu aktuellen Geräten passt weiss ich nicht

**Reply by fischli** [reply]
Ich vermute mal das die Infos auf dieser Seite für aktuelle Versionen nicht mehr brauchbar sind. Das referenzierte Tool cpmaccfg findet sich auch nirgendwo mehr.

**Reply by fda77** [reply]
Im Kernel könnte es noch immer ein "cpmac" geben, der Source liegt unter source/
